.env_SAMPLE

```diff
@@ -45,23 +45,22 @@ export ALLOW_ADMIN_URL=True
 #export AKAMAI_FAST_PURGE_URL=<akamai_fast_purge_url>
 #export AKAMAI_PURGE_ALL_URL=<akamai_purge_all_url>
 
-########################
-# MySQL Database Server.
-########################
+#########################################################################
+# Django database.
+#
+# See https://github.com/kennethreitz/dj-database-url for URL formatting.
+#########################################################################
+
+export DATABASE_URL=sqlite:///db.sqlite3
```

Comment:
Default value when using `.env` is SQLite because that makes it easier for new users of this repo to get up and running without having to set up MySQL or Postgres.